frontend/src/pages/Interventions/Activities.tsx

```diff
@@ -122,12 +122,9 @@ const Activities: FunctionComponent<ActivityProps> = ({
           let durationInSeconds =
             (curDate.getTime() - activityStartTime.getTime()) / 1000;
           setActivityStartTime(curDate);
-          if (
-            currentWordIdx === wordList.length - 1 &&
-            nextActivityIdx === 0
-          ) {
-            history.push(`/interventions/reward`);
+          if (nextWordIdx === 0 && nextActivityIdx === 0) {
```

Comment:
what's the reasoning behind this change? (not saying it's wrong, just curious

Reply:
wanted to be consistent lol, i think it's basically the same calculation but since we're using nextActivityIndex i wanted to use nextWordIndex for clarity. if you're referring to the line swap below this it's because i initially thought the finishedIntervention call was being interrupted by the history.push call (it wasn't)

Reply:
ahh i see

also I said I was gonna test this today, but the api key expired? so can't really do much